### PR TITLE
Enable sass source map, remove compass and ruby

### PR DESCRIPTION
**Pull request by daigofuji**
Inspired by awesome PR #7 by @atogle I've attempted to remove all references from compass and also enable source map in sass by using newer version of sass. 

Still need to test in other computer/clean install to make sure there is no ill effect.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 source 'http://rubygems.org'
 
-gem 'compass', '~> 0.12.7'
+gem 'sass', '~> 3.4.9'
```
Comment (atogle) [inline Gemfile]: What do you use the sass gem for?

Comment (daigofuji) [inline Gemfile]: OK, I guess I am not understanding it right. So, this means I don't need ruby at all? No `bundle install` required? https://github.com/HBRGTech/daisy/blob/master/puppet/manifests/site.pp#L37-L58

**Comment by atogle** [comment]
If compass was the only thing using ruby, then yes. `grunt-sass` uses `node-sass` which uses `libsass`, written in C++. More here: http://benfrain.com/lightning-fast-sass-compiling-with-libsass-node-sass-and-grunt-sass/

Did we just eliminate your ruby dependency?

**Comment by daigofuji** [comment]
I think you did. I did a lot of testing/reading over the weekend and I found (knock on wood) no ill-effect. Love it.

**Comment by daigofuji** [comment]
This one is tested and ready to merge. @flagster76 can you do a quick check?

**Comment by flagster76** [comment]
Sure!  I'll take a look and try it out before the day is out.  Thanks!

**Comment by flagster76** [comment]
Ran without an issue for me, either as a pull on existing or fresh clone.  Great stuff.

**Pull request merged**